nxcomp/configure.ac

```diff
@@ -74,6 +74,30 @@ else
   CPPFLAGS="$CPPFLAGS -DIN_ADDR_T=in_addr_t"
 fi
 
+AC_ARG_WITH(info,
+            [AS_HELP_STRING([--with-info],[define -DINFO at compile time to get info session log output])])
+
+AC_ARG_WITH(valgrind,
+            [AS_HELP_STRING([--with-valgrind],[define -DVALGRIND to enable valgrind memory checker workarounds])])
+
+dnl Check whether --with-info or --without-info was given.
+
+if test "${with_info}" = yes; then
+  echo -e "enabling info output in the session log file"
+  CPPFLAGS="$CPPFLAGS -DINFO"
```

Comment:
> AC_ARG_ENABLE

Converted -with-info and -with-valgrind to --enable-debug and --enable-valgrind now.